tests/utils/create-e2e-fixture.ts

```diff
@@ -542,12 +542,18 @@ export const fixtureFactories = {
       packagePath: 'apps/next-app',
       buildCommand: 'nx run next-app:build',
       publishDirectory: 'dist/apps/next-app/.next',
+      env: {
+        NX_ISOLATE_PLUGINS: 'false',
+      },
     }),
   nxIntegratedDistDir: () =>
     createE2EFixture('nx-integrated', {
       packagePath: 'apps/custom-dist-dir',
       buildCommand: 'nx run custom-dist-dir:build',
       publishDirectory: 'dist/apps/custom-dist-dir/dist',
+      env: {
+        NX_ISOLATE_PLUGINS: 'false',
+      },
```

Comment:
nx on next canary starting failing with errors mentioned in https://github.com/nrwl/nx/issues/27040 (but it's not exactly the same, because issue mention `postinstall`, while for us errors were happening at `next build` time, so despite that issue being closed with a fix, I think we are hitting similar problem in different place). I tried `NX_ISOLATE_PLUGINS` which was suggested as workaround there and that seemed to work here as well